streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignor.java

```diff
@@ -277,22 +285,32 @@ public Subscription subscription(Set<String> topics) {
      * 3. within each client, tasks are assigned to consumer clients in round-robin manner.
      */
     @Override
-    public Map<String, Assignment> assign(Cluster metadata, Map<String, Subscription> subscriptions) {
+    public Map<String, Assignment> assign(final Cluster metadata,
```

Comment:
Could we add a unit test with two subscriptions, v1 and v2, and then check that the returned assignment map contains v1?